### PR TITLE
module 3 assignment

### PR DESCRIPTION
**Pull request by JuleeGunjal**

### CODE REVIEW

File: module_3/assignment1_2.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+let age = 25;
+
+switch (true){
+  case (age <= 12):
+    console.log("child");
+    break;
+  case (18 <= age >= 13):
```
Comment (tejashreeSurveJosh) [inline module_3/assignment1_2.js]: Please check this will not work condition is wrong.

File: module_3/assignment1_3.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// 3. Declare a variable let arraySize = 25;. Using a for loop, add numbers from 
+// one onwards into an array till the arraySize is reached.
+
+let arraySize = 25;
+let array = []; 
+
+for(let i = 0; i < arraySize;){
+  array[i] = i;
```
Comment (tejashreeSurveJosh) [inline module_3/assignment1_3.js]: you can also use push method

File: module_3/assignment1_5.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+}
+// Output => 0 1 2
+
+console.log("Using Break array is: ",array); // Using Break array is:  [ 0, 1, 2 ]
+
+for(let i=0;i<5;i++){
+  console.log(i)
+  array[i] = i;
+  if(i==2)
+  {
```
Comment (tejashreeSurveJosh) [inline module_3/assignment1_5.js]: If there is only 1 statement for any loop, then braces is not required directly return statement you can use 
if (i == 2) return;

File: module_3/assignment1_2.js (hunk 1 above)
Comment (JuleeGunjal) [reply module_3/assignment1_2.js]: Done

File: module_3/assignment1_3.js (hunk 2 above)
Comment (JuleeGunjal) [reply module_3/assignment1_3.js]: DOne

File: module_3/assignment1_5.js (hunk 3 above)
Comment (JuleeGunjal) [reply module_3/assignment1_5.js]: Done

**Pull request merged**